### ISSUE TITLE
Mount speed

### ISSUE THREAD

**Issue by ShadoMaso** [IoIxD/Cobblemounts#31]

**Comment by IoIxD** [comment]
And you are using the recent version of the config? You need to remove whatever config you had before if you're upgrading.

**Comment by ShadoMaso** [comment]
I didnt touch the config, nor upgrading, just downloaded both mod for the first time and that's it

**Comment by IoIxD** [comment]
Can you record a video?

**Comment by ShadoMaso** [comment]
here you go

https://www.youtube.com/watch?v=gkZA2DztBu4

any Mon I tested have similar speed

**Comment by dennorske** [comment]
I have the same issue, but it only affects ground movement. Flying and swimming is fine

**Comment by ShadoMaso** [comment]
can confirm I barely tested that before but flying and swimming scaling are good

**Comment by otterpawps** [comment]
Just updated and have the same issue.

**Comment by IoIxD** [comment]
> here you go
> 
> https://www.youtube.com/watch?v=gkZA2DztBu4
> 
> any Mon I tested have similar speed

This is with a level 1 Pokemon. Level 1 Pokemon being extremely slow is intended behavior.

**Comment by IoIxD** [comment]
> here you go
> 
> https://www.youtube.com/watch?v=gkZA2DztBu4
> 
> any Mon I tested have similar speed

What does your config look like?

**Comment by ShadoMaso** [comment]
> > here you go
> > https://www.youtube.com/watch?v=gkZA2DztBu4
> > any Mon I tested have similar speed
> 
> What does your config look like?

no idea how to even check the config, I just downloaded the lastest version on Modrinth

**Comment by meiozei** [comment]
hello! to check the config just look for minecraft/config folder and look for cobblemounts.toml 

by default, it's empty and you can set whatever pokemon you can mount and the flying and swimming speed. there is a config pastebin on the modrinth page although its heavily outdated and you can use that for reference.

**Comment by ShadoMaso** [comment]
that's the config I have 

![Screenshot 2024-05-24 161907](https://github.com/IoIxD/Cobblemounts/assets/147556486/6e28a500-b948-4414-8ddf-2c7983360b25)

**Comment by IoIxD** [comment]
> that's the config I have 
> 
> ![Screenshot 2024-05-24 161907](https://github.com/IoIxD/Cobblemounts/assets/147556486/6e28a500-b948-4414-8ddf-2c7983360b25)
> 

Yeah that's the old config. Delete it so the mod can regenerate it.

**Comment by ShadoMaso** [comment]
well that fix it, I know I used some modpack before that had the mod it might have mixed it up but that was my first time using it in a manual download way
